docs/imm-otel.md

```diff
@@ -0,0 +1,101 @@
+
+The following sections describe each step in detail.
+
+## Deploy the SignalFx Smart Agent
+
+The SignalFx Smart Agent receives metrics from [integrations](https://docs.signalfx.com/en/latest/integrations/integrations-reference/index.html “Integrations reference guide”) before exporting them to the OpenTelemetry Collector for central management and processing. Monitors are one of the main SignalFX Smart Agent components. Monitors gather metrics from the host and from running applications. See [monitor configuration](https://docs.signalfx.com/en/latest/integrations/agent/monitors/_monitor-config.html “Monitor Configuration”) for configuration options.  
+
+**Step 1: Install the SignalFx Smart Agent**
+
+There are two options for installing the SignalFx Smart Agent:
+- Use the [quick installation](https://docs.signalfx.com/en/latest/integrations/agent/quick-install.html "Quick Install") option for simplified SignalFx Smart Agent command-line installation on a single host.
+- Use the [advanced installation](https://docs.signalfx.com/en/latest/integrations/agent/advanced-install-options.html "Advanced Installation Options") option for bulk deployments and for configuring various monitors for your environment.
+
+**Step 2: Configure the SignalFx Smart Agent**
+
+The SignalFx Smart Agent is configured primarily by a YAML document located at `/etc/signalfx/agent.yaml`. The location of the configuration file can be specified by the `-config` flag to the agent binary (`signalfx-agent`).
+
+See [Agent Configuration](https://docs.signalfx.com/en/latest/integrations/agent/config-schema.html "Agent Configuration") for configuration options and an example configuration file. See [deployments](https://github.com/signalfx/signalfx-agent/tree/master/deployments "Deployments") for more information on the various ways to deploy the SignalFx Smart Agent.
+
+## Deploy an OpenTelemetry Collector for SignalFx
+
+After installing the SignalFx Smart Agent on each host, you can deploy an OpenTelemetry Collector in each datacenter/region/cluster where traced applications run. In general, an OpenTelemetry Collector should receive data from the SignalFx Smart Agent.
+
+The OpenTelemetry Collector uses pipelines to receive, process, and export trace data with components conveniently known as receivers, processors, and exporters. Set up pipelines with services. You can also add extensions that provide an OpenTelemetry Collector with additional functionality, such as diagnostics and health checks. The OpenTelemetry Collector has two versions: a [core version](https://github.com/open-telemetry/opentelemetry-collector "Core Version") and a [contributions version](https://github.com/open-telemetry/opentelemetry-collector-contrib "Contributions"). The core version provides receivers, processors, and exporters for general use. The contributions version provides receivers, processors, and exporters for specific vendors and use cases.
```

Comment:
I am not sure if the customer cares -- if they do, this info is already covered in the Otel repo so perhaps just link so you do not need to keep updating this